includes/Admin/AnyCommentGenericSettings.php

```diff
@@ -791,7 +791,7 @@ public function init_settings () {
 				     $this->field_builder()
 				          ->checkbox()
 				          ->set_id( self::OPTION_LINKS_ON_HOLD )
-				          ->set_title( __( 'Links on Hold', "anycomment" ) )
+				          ->set_title( __( 'Hold all comments with links', "anycomment" ) )
```

Comment:
We have titles capitalized first letter, maybe we could make it the same way here, so that we won't need to fix it for many languages later

Reply:
Agreed, but in places where the text is user-facing we should aim for proper capitalization (which i think is true in most places).

Reply:
```suggestion
				          ->set_title( __( 'Hold All Comments with Links', "anycomment" ) )
```